packages/tiptap-extensions/src/extensions/History.js

```diff
@@ -23,7 +30,10 @@ export default class History extends Extension {
 
   get plugins() {
     return [
-      history(),
+      history({
+        depth: this.config.depth,
```

Comment:
It's **`this.options.depth`** not `this.config.depth`

Reply:
@Chrissi2812 Whoops! Fixed! :smiley: